doc/release.md

```diff
@@ -0,0 +1,78 @@
+# Release
+
+**JVS-PLUGIN-GITHUB is not an official Google product.**
+
+We leverage [goreleaser](https://goreleaser.com/) for SCM (GitHub) and image
+release.
+
+## New Release
+
+- If a new JVS base image is needed, update [Dockerfile](https://github.com/abcxyz/jvs-plugin-github/blob/main/Dockerfile)
+to point to a new JVS base image, and merge the PR.
+
+- Send a PR to update all dependencies For Go
+```sh
+go get -u && go mod tidy
+```
+
+- Create a tag using [create-tag workflow](https://github.com/abcxyz/jvs-plugin-github/blob/main/.github/workflows/create-tag.yml) on default branch and
```

Comment:
https://github.com/abcxyz/jvs-plugin-github/actions/workflows/create-tag.yml is better

Reply:
Done